ballerina/Ballerina.toml

```diff
@@ -5,8 +5,8 @@ name = "paypal.orders"
 version = "2.0.0"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = []
-# icon = "icon.png" # TODO: update icon.png
+keywords = [] # TODO: Add keywords
+# icon = "icon.png" # TODO: Add icon
```

Comment:
Let's add the icon and the keywords (in `build-config/resources/Ballerina.toml` file. Don't directly edit this file.

Possible keywords are ["Paypal", "Payments"]